po/data.table.pot

```diff
@@ -152,11 +152,6 @@ msgstr ""
 msgid "Assigning to %d row subset of %d rows\n"
 msgstr ""
 
-#: assign.c:326
-#, c-format
-msgid "Added %d new column initialized with all-NA\n"
-msgstr ""
-
 #: assign.c:332
```

Comment:
This seems to be missing the singular forms of the translated messages from the `Pl_` macro. @MichaelChirico, what's the right `xgettext` call to capture them?